### PR TITLE
.rultor.yml: Install dependencies

### PR DESCRIPTION
**Pull request by Nosferatul**
Added apt-get update, apt-get install, pip3 install.

Fixes https://github.com/coala-analyzer/coala/issues/1904

### CODE REVIEW

File: .rultor.yml [hunk 1]
```diff
@@ -16,3 +19,4 @@ release:
 
 decrypt:
   rultor_secrets.sh: "repo/.misc/rultor_secrets.sh.asc"
+
```
Comment (Makman2) [inline .rultor.yml]: unneeded newline

**Comment by Makman2** [comment]
and please rebase instead of using merge commits :)

**Comment by Nosferatul** [comment]
My bad, I don't know yet how to use properly git. I will open another pr.

**Comment by Makman2** [comment]
Please dont close and re open PRs if you make changes, just rebase them^^
If you need help with that ping us^^

**Pull request closed without merging**